src/services/recim/activity.ts

```diff
@@ -59,7 +59,7 @@ const getActivityByID = (ID: number): Promise<Activity> => http.get(`recim/activ
 
 const getActivities = (active: boolean, time: String): Promise<Activity[]> => {
   if (time) return http.get(`recim/activities?active=${active}&time=${time}`);
-  if (active) return http.get(`recim/activities?active=${active}`);
+  if (typeof active !== 'undefined') return http.get(`recim/activities?active=${active}`);
```

Comment:
This creates a lot of confusion in this function. Active is a boolean operator, so with this current implementation, if active is false, it will only get the active activities.

Reply:
this was not my implementation. This is from @cpabbot's PR.